### ISSUE TITLE
My systems dont wok!

### ISSUE THREAD

**Issue by jp-polito** [PartPilotLab/PartPilot#24]

**Comment by LenniM** [comment]
Hey @jp-polito, thanks for raising the issue. PartPilot's autocomplete is currently only integrated with LCSC-parts. Can it be, that the product code isn't from LCSC? If so, what distributor are you using?

**Comment by jp-polito** [comment]
Thank you very much for the answer! Yes, I am using an LCSC value, this code X116241354 was read from the barcode that comes in the LCSC component package.

**Comment by LenniM** [comment]
Okay, strange... I am not able to find the part inside the LCSC website. Maybe they have deleted the part's record from their database. The only thing I can suggest to you right now is to manually type in the details. 
![not-found](https://github.com/user-attachments/assets/7b71dfa5-ef83-490c-af4e-4b593da9dfb9)

**Comment by jp-polito** [comment]
Yes! If I enter the code C2833372 manually it works but the code read by the bar code does not work... I tested it by reading the value of the QR code too.
